### ISSUE TITLE
Broken in node 8

### ISSUE THREAD

**Issue by MiniGod** [agirorn/jasmine-slow-reporter#3]
I keep getting an error when running this in node 8. Other versions seam to be working fine for me.

```
/path/to/repo/node_modules/jasmine-slow-reporter/lib/find-spec-file.js:13
  return line.filename.match('.spec.js');
                      ^

TypeError: Cannot read property 'match' of null
    at isSpec (/path/to/repo/node_modules/jasmine-slow-reporter/lib/find-spec-file.js:13:23)
    at /path/to/repo/node_modules/underscore/underscore.js:227:11
    at Function._.each._.forEach (/path/to/repo/node_modules/underscore/underscore.js:153:9)
    at _.filter._.select (/path/to/repo/node_modules/underscore/underscore.js:226:7)
    at findSpecFile (/path/to/repo/node_modules/jasmine-slow-reporter/lib/find-spec-file.js:9:16)
    at Env.jasmineSlowReporterIt [as it] (/path/to/repo/node_modules/jasmine-slow-reporter/lib/it-factory.js:10:40)
    at addSpecsToSuite (/path/to/repo/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1113:14)
    at Env.describe (/path/to/repo/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
    at describe (/path/to/repo/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
    at Object.<anonymous> (/path/to/repo/spec/unit/barcode.spec.js:15:1)
```

**Comment by agirorn** [comment]
Does this happen all of the time?
If not can you then give me a more detailed example?

**Comment by agirorn** [comment]
Thanks for reporting the bug. Is should be fixed by bad51401e31b68075fbce2e72350167db32f4eaf and published in version v0.1.0
